Realm/RLMAppCredentials_Private.hpp

```diff
@@ -0,0 +1,31 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2020 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#ifndef RLMAppCredentials_Private_hpp
+#define RLMAppCredentials_Private_hpp
+
+#import "RLMAppCredentials.h"
+#import "sync/app_credentials.hpp"
+
+@interface RLMAppCredentials()
+
+@property std::shared_ptr<realm::app::AppCredentials> appCredentials;
```

Comment:
There doesn't appear to be any reason for this to be a shared_ptr. It should also be marked as `nonatomic`.